build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
     id 'idea'
     id 'java'
-    id 'org.jetbrains.intellij' version '1.9.0'
+    id 'org.jetbrains.intellij' version '1.10.0'
```

Comment:
Should be reverted as it breaks the MacOS build

Reply:
I'm not going to merge this as it doesn't work with 1.9 for me. I'm trying to figure out why 1.10 is failing. Same for tekton